dist/svg/ds4/icons.svg

```diff
@@ -573,12 +573,12 @@
     </symbol>
     <symbol id="icon-checkbox-mixed" viewBox="0 0 18 18">
-        <path d="M6.4 17H11.6C12.7366 17 13.5289 16.9992 14.1458 16.9488C14.7509 16.8994 15.0986 16.8072 15.362 16.673C15.9265 16.3854 16.3854 15.9265 16.673 15.362C16.8072 15.0986 16.8994 14.7509 16.9488 14.1458C16.9992 13.5289 17 12.7366 17 11.6V6.4C17 5.26339 16.9992 4.47108 16.9488 3.85424C16.8994 3.24907 16.8072 2.90138 16.673 2.63803C16.3854 2.07354 15.9265 1.6146 15.362 1.32698C15.0986 1.19279 14.7509 1.10062 14.1458 1.05118C13.5289 1.00078 12.7366 1 11.6 1H6.4C5.26339 1 4.47108 1.00078 3.85424 1.05118C3.24907 1.10062 2.90138 1.19279 2.63803 1.32698C2.07354 1.6146 1.6146 2.07354 1.32698 2.63803C1.19279 2.90138 1.10062 3.24907 1.05118 3.85424C1.00078 4.47108 1 5.26339 1 6.4V11.6C1 12.7366 1.00078 13.5289 1.05118 14.1458C1.10062 14.7509 1.19279 15.0986 1.32698 15.362C1.6146 15.9265 2.07354 16.3854 2.63803 16.673C2.90138 16.8072 3.24907 16.8994 3.85424 16.9488C4.47108 16.9992 5.26339 17 6.4 17ZM0.435974 15.816C0 14.9603 0 13.8402 0 11.6V6.4C0 4.15979 0 3.03969 0.435974 2.18404C0.819467 1.43139 1.43139 0.819467 2.18404 0.435974C3.03969 0 4.15979 0 6.4 0H11.6C13.8402 0 14.9603 0 15.816 0.435974C16.5686 0.819467 17.1805 1.43139 17.564 2.18404C18 3.03969 18 4.15979 18 6.4V11.6C18 13.8402 18 14.9603 17.564 15.816C17.1805 16.5686 16.5686 17.1805 15.816 17.564C14.9603 18 13.8402 18 11.6 18H6.4C4.15979 18 3.03969 18 2.18404 17.564C1.43139 17.1805 0.819467 16.5686 0.435974 15.816Z" />
-        <rect x="4" y="8" width="10" height="2" rx="1" />
+        <rect width="18" height="18" rx="4" />
+        <rect x="4" y="8" width="10" height="2" fill="var(--checkbox-mixed-foreground, #ffffff)"/>
```

Comment:
I don't think this is needed together with the CSS style.
I think this should only be needed when there are two different colors on the icon. Otherwise the fill should be pulled from the `color`

Reply:
The checkbox has the background color as well as the fill color here - there are two colors - because of the way they are outlined, you do need the fill here because otherwise the background covers up the horizontal dash on the icon.

Reply:
Okay so I see what you mean.
I think this is fine. However there is another part of the code that's not needed.